BAZProject/Common/Networking/Models/MovieDetailDataObject.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  MovieDetailDataObject.swift
+//  BAZProject
+//
+//  Created by gescarcega on 05/05/23.
+//
+
+import Foundation
+
+struct MovieDetailDataObject: Decodable, Equatable {
+    
+    var overview: String?
+    var recommendations: GenericResponseDataObject?
+    var reviews: ReviewCollectionDataOject?
+    var similar: GenericResponseDataObject?
+    var credits: CastDataObject?
```

Comment:
Hay alguna razón por la cual no podemos dejar las propiedades como constantes?
```suggestion
    let overview: String?
    let recommendations: GenericResponseDataObject?
    let reviews: ReviewCollectionDataOject?
    let similar: GenericResponseDataObject?
    let credits: CastDataObject?
```

Misma sugerencia para las propiedades en

- `PersonDetailDataObject`
- `PersonCastingDataObject`
- `ReviewCollectionDataObject`
- `ReviewDataObject`
- `TVDetailDataObject`